vit-servicing-station-lib/migrations/2020-05-22-112032_setup_db/up.sql

```diff
@@ -30,12 +30,18 @@ create table proposals
     proposer_url VARCHAR NOT NULL,
     proposer_relevant_experience VARCHAR NOT NULL,
     chain_proposal_id BLOB NOT NULL,
-    chain_proposal_index BIGINT NOT NULL,
     chain_vote_options VARCHAR NOT NULL,
-    chain_voteplan_id VARCHAR NOT NULL,
     challenge_id INTEGER NOT NULL
 );
 
+create table proposals_voteplans (
+    proposal_id VARCHAR NOT NULL primary key,
+    chain_voteplan_id VARCHAR NOT NULL,
+    chain_proposal_index BIGINT NOT NULL,
```

Comment:
does this really need to be bigint? I think we only support 256 proposals in a voteplan

Reply:
I guess not, I didn't really change it though, it was already a bigint. 

That said, I think it makes no difference in sqlite, since they say

```
INTEGER. The value is a signed integer, stored in 0, 1, 2, 3, 4, 6, or 8 bytes depending on the magnitude of the value.
```
and `BIGINT` has `INTEGER` affinity/storage class.

I could change it anyway though.